apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns-from-api.php

```diff
@@ -252,7 +252,7 @@ private function reregister_core_patterns() {
 				// Gutenberg registers patterns with varying prefixes, but categorizes them using `core/*` in a blockTypes array.
 				// This will ensure we remove `query/*` blocks for example.
 				// TODO: We need to revisit our usage or $pattern['blockTypes']: they are currently an experimental feature and not guaranteed to reference `core/*` blocks.
-				$pattern_block_type_or_name = ! empty( $pattern['blockTypes'][0] ) ? $pattern['blockTypes'][0] : $pattern['name'];
+				$pattern_block_type_or_name = is_array( $pattern['blockTypes'] ) && ! empty( $pattern['blockTypes'][0] ) ? $pattern['blockTypes'][0] : $pattern['name'];
```

Comment:
I _think_ this might need to be an `isset` instead of an `is_array`?

Reply:
Defo. 
👍